modules/core/src/lib/deck.js

```diff
@@ -59,7 +60,9 @@ function getPropTypes(PropTypes) {
     layerFilter: PropTypes.func,
     views: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
     viewState: PropTypes.object,
-    effects: PropTypes.arrayOf(PropTypes.instanceOf(Effect)),
+    effects: PropTypes.arrayOf(
+      PropTypes.oneOfType([PropTypes.instanceOf(Effect), PropTypes.instanceOf(LightingEffect)])
```

Comment:
https://reactjs.org/docs/typechecking-with-proptypes.html

`LightingEffect` inherits from `Effect`, `instanceof` returns true if an object belongs to the `class` or a `class` inheriting from it.

 So you don't need to specify `PropTypes.instanceOf(LightingEffect)` here.